index.html

```diff
@@ -41,5 +41,161 @@ <h1>I’m Mariane Glad to see you!</h1>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+
+      <div class="work-card">
+        <img src="./assets/work-2.png" alt="work image" />
+        <div class="details">
+          <h2>Multi-Post Stories</h2>
+          <ul class="subTitle">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="langauges">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+
+      <div class="work-card">
+        <img src="./assets/work-3.png" alt="work image" />
+        <div class="details">
+          <h2>Tonic</h2>
+          <ul class="subTitle">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="langauges">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+
+      <div class="work-card">
+        <img src="./assets/work-4.png" alt="work image" />
+        <div class="details">
+          <h2>Multi-Post Stories</h2>
+          <ul class="subTitle">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="langauges">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
+      <div class="info">
+        <h1>About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <a href="#">LET’S CONNECT</a>
+        <ul class="socials">
+          <li>
+            <img src="./assets/github.svg" alt="github icon" />
+          </li>
+          <li>
+            <img src="./assets/Linkedin.svg" alt="Linkedin icon" />
+          </li>
+          <li>
+            <img src="./assets/peace.svg" alt="peace icon" />
+          </li>
+          <li>
+            <img src="./assets/twitter.svg" alt="twitter icon" />
+          </li>
+          <li>
+            <img src="./assets/meduim.svg" alt="medium icon" />
+          </li>
```

Comment:
- [ ] Each icon should be a link. this makes it easy for companies or recruiters to connect with you